content/docs/how-we-work/security.md

```diff
@@ -1,37 +1,33 @@
 ---
 title: Security
 weight: 3
-description: Security is an everlasting concern in the world of software engineering. Understanding what security means in the context of our work and the various vectors that might be responsible for compromising our system's security is extremely important.
+description: Security is an everlasting concern in software engineering. Understanding security in the context of our work and recognizing various vectors that might compromise our system’s security is crucial. Security is a broad and deep topic, and it’s a team’s responsibility to implement safeguards at various levels.
 ---
 
 # Security
 
-Security is an everlasting concern in the world of software engineering. Understanding what security means in the context of our work and the various vectors that might be responsible for compromising our system's security is extremely important.
+Security is an everlasting concern in software engineering. Understanding security in the context of our work and recognizing various vectors that might compromise our system’s security is crucial. Security is a broad and deep topic, and it’s a team’s responsibility to implement safeguards at various levels.
 
-That said, security is a wide and deep topic, and a single person cannot hope to span the width and depth of this challenging area. It is the team's responsibility to understand security at various levels and implement safeguards.
+## Credentials Management
 
-For example, an application relies on database credentials to connect to the database. There can be API keys with which the application communicates with the third-party service. They must not be kept in the repository. There should not even be a Git commit referring to those credentials. They can be misused if a malicious user gets hold of them. Use environment variables or a vault for storing and retrieving them.
+Applications rely on database credentials and API keys to communicate with third-party services. These must not be kept in the repository or committed in Git history as they can be misused if exposed. Use environment variables or a vault for storing and retrieving them.
```

Comment:
The phrase "For example" was necessary here. Applications don't _always_ rely on database credentials and API keys. There may be other secrets such as private keys, certificates, etc. We don't need to mention all of them and that is why there was this qualifier: "For example".